### PR TITLE
Location reassignment name format changes

### PR DESCRIPTION
**Pull request by mkangia**
https://dimagi-dev.atlassian.net/browse/ICDS-1464

##### SUMMARY
For awc and supervisor level the names for locations on HQ are present as a concatenation of the raw name and site code as "Location Name [location site code]"
This PR addresses that change so that users are not confused when they download a template and when the users upload information for new locations to be created, the concatenation is done here as well.

##### FEATURE FLAG
`LOCATION_REASSIGNMENT`

### CODE REVIEW

File: custom/icds/location_reassignment/utils.py [hunk 1]
```diff
@@ -98,3 +98,6 @@ def split_location_name_and_site_code(name):
         return match.groups()
     return name, None
 
+
+def append_location_name_and_site_code(name, site_code):
+    return f"{name.rstrip()} [{site_code}]"
```
Comment (orangejenny) [inline custom/icds/location_reassignment/utils.py:103]: Why just `rstrip`?

**Review by orangejenny — APPROVED** [review]

File: custom/icds/location_reassignment/utils.py (hunk 1 above)
Comment (mkangia) [reply custom/icds/location_reassignment/utils.py:103]: just trying to handle the case where someone left an empty space in the end of the name and avoid cases like "location name    [123]". If they left it in the beginning, that should be okay

Comment (mkangia) [reply custom/icds/location_reassignment/utils.py:103]: I could not put multiple spaces in the comment, but i wanted to put something like
location name (extra spaces more than one) [location code]

Comment (mkangia) [reply custom/icds/location_reassignment/utils.py:103]: @orangejenny let me know if this is cool, feel free to merge if it is or drop a note, this is g2g.

Comment (orangejenny) [reply custom/icds/location_reassignment/utils.py:103]: Sounds good to me, merging.

**Pull request merged**